### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by nullpixel**

### CODE REVIEW

File: Commands/Commands/uservoice.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 var commands = []
 
-var checker = require('../../Utils/access_checker')
+//var checker = require('../../Utils/access_checker')
```
Comment (Arzte) [inline Commands/Commands/uservoice.js:3]: Why?

File: Commands/Commands/utils.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 var commands = []
 
-var checker = require('../../Utils/access_checker')
-var logger = require('../../Utils/error_loggers')
+//var checker = require('../../Utils/access_checker')
+//var logger = require('../../Utils/error_loggers')
```
Comment (Arzte) [inline Commands/Commands/utils.js:4]: Why?

File: Commands/Commands/uservoice.js (hunk 1 above)
Comment (nullpixel) [reply Commands/Commands/uservoice.js:3]:
```/Users/jamiebishop/development/DiscordFeedback/Commands/Commands/uservoice.js
  3:5  error  'checker' is assigned a value but never used  no-unused-vars

/Users/jamiebishop/development/DiscordFeedback/Commands/Commands/utils.js
  3:5  error  'checker' is assigned a value but never used  no-unused-vars
  4:5  error  'logger' is assigned a value but never used   no-unused-vars```

**Comment by LW001** [comment]
ah this is where that was commented out, tyvm https://travis-ci.org/SteamingMutt/DiscordFeedback/builds/246622786

**Pull request merged**